src/qt/qt_mainwindow.cpp

```diff
@@ -1646,6 +1652,9 @@ void MainWindow::blitToWidget(int x, int y, int w, int h, int monitor_index)
 
 void MainWindow::keyReleaseEvent(QKeyEvent* event)
 {
+    if (event->key() == Qt::Key_Pause) {
+        plat_pause(dopause ^ 1);
```

Comment:
Wouldn't this prevent the use of the Pause key inside the emulated machine?  
(Although this used to be a problem in the Win32 UI as well...)

Reply:
Pause key isn't emulated.

Reply:
What do you mean? It works for me.

Reply:
Maybe only on Windows, but not on anything else.

Reply:
Then shouldn't *that* be considered a bug, and fixed?

Reply:
More like the pause key should perform only a single function at a time, not both.

Reply:
Then the key to pause the emulator should be (and should have been ever since a long time ago) reassigned to a different key combination.

Reply:
We would need to figure out one then.

Reply:
> Then the key to pause the emulator should be (and should have been ever since a long time ago) reassigned to a different key combination.

This could be an option (switch).

Reply:
I'm going to merge this now because it fixes an open issue. The pause button behavior can be fixed in the future.